pageserver/src/page_service.rs

```diff
@@ -2567,6 +2575,13 @@ where
 
         fail::fail_point!("ps::connection-start::process-query");
 
+        if let Some((app_name, compute_type)) = &self.application_info {
+            tracing::Span::current().record("application", field::display(app_name));
```

Comment:
`application` is wrong. The [field is defined here](https://github.com/neondatabase/neon/blob/b86c4fabb45d91b570ed308f486e2ee1e0684eb7/pageserver/src/page_service.rs#L240)


```suggestion
            tracing::Span::current().record("application_name", field::display(app_name));
```

Reply:
Also, the `Span::current().record()` will be a no-op here because the `application_name` field is specififed in the parent span.